code/game/machinery/guestpass.dm

```diff
@@ -58,6 +58,9 @@
 	my_terminal_id = ++global_terminal_id
 
 /obj/machinery/computer/guestpass/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/card/id/nct_data_chip))
+		to_chat(user, "<span class='warning'>The [I] does not seem compatible with this terminal!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>[I] does not seem compatible with this terminal!</span>")
```
Otherwise it'll say `The the ID`